runtime/doc/news.txt

```diff
@@ -169,7 +169,8 @@ STARTUP
 
 TERMINAL
 
-• todo
+• |nvim_open_term()| can be called with a non-empty buffer. The buffer
+  contents are piped to the PTY and displayed as terminal output.
```

Comment:
Great idea. This likely will help with https://github.com/neovim/neovim/issues/28297